File: 241121/청소당번 정하기/cleaning-numbering.c
#include <stdio.h>

int main() {
    // 여기에 코드를 작성해주세요.

    int n;

    int cnt_c = 0;
    int cnt_h = 0; 
    int cnt_t = 0;
    
    scanf("%d", &n);
    
  

    for (int i=1; i<=n; i++){
        if (i%12==0){
            cnt_t++;
        }
        else if (i%3==0){
            cnt_h++;
        }
        else if (i%2==0){
            cnt_c++;
        }
    }

    printf("%d %d %d", cnt_c, cnt_h, cnt_t);
    return 0;
}

File: 241127/청소당번 정하기/cleaning-numbering.py
"""
12 t
3 h
2 c
"""
cnt_c = 0
cnt_h = 0
cnt_t = 0

n = int(input())

for i in range(1, n+1) :
    if (i%12 == 0):
        cnt_t+=1
    elif(i%3 == 0):
        cnt_h+=1
    elif(i%2==0) :
        cnt_c+=1

print(f"{cnt_c} {cnt_h} {cnt_t}")